procedures/unit-testing-tracing-analytics.ipf

```diff
@@ -32,4 +40,43 @@ static Function/WAVE GetTotals()
 	return totals
 End
 
+static Structure CollectionResult
+	WAVE/T functions
+	WAVE lines
+	WAVE calls
+	WAVE sums
+	variable count
+EndStructure
+
+static Function CollectFunctions(WAVE totals, WAVE/T procs, STRUCT CollectionResult &result)
+	variable i, j, startIndex, endIndex
+	variable procCount = DimSize(procs, UTF_ROW)
+	variable funcCount = GetMaxFuncCount()
+	DFREF dfr = GetPackageFolder()
+	WAVE/WAVE funcLocations = UTF_Tracing#GetFuncLocations()
+	variable lbFuncList = FindDimLabel(funcLocations, UTF_COLUMN, "FUNCLIST")
+	variable lbFuncStart = FindDimLabel(funcLocations, UTF_COLUMN, "FUNCSTART")
+
+	Make/FREE=1/N=(procCount, funcCount)/T result.functions
+	Make/FREE=1/N=(procCount, funcCount) result.lines
```

Comment:
Make also supports a comma separated list as arguments to create multiple waves with the same properties with one call.
e.g. `Make/FREE=1/N=(procCount, funcCount) result.lines, result.calls, result.sums`